#402 documentation for cleaning up nodes results in errors (#633)
rancher/rancher-docs: docs/how-to-guides/new-user-guides/manage-clusters/clean-cluster-nodes.md

<Tabs>
<TabItem value="RKE1">

Before you run the following commands, first remove the node through the Rancher UI.

To remove a node:

> 1. Click the **☰** and select **Cluster Management**.

Reviewer: ```suggestion
1. Click **☰** and select **Cluster Management**.
```

1. In the table of clusters, click the name of the cluster the node belongs to.
1. In the **Machine Pool** tab, click the checkbox next to the node's state.
1. Click **Delete**.

If you remove the entire cluster instead of an individual node, or skip rermoving the node through the Rancher UI, follow these steps:

1. [Remove](#docker-containers-images-and-volumes) the Docker containers from the node and [unmount](#mounts) any volumes.
1. Reboot the node.
1. [Remove](#directories-and-files) any remaining files.
1. Confirm that network interfaces and IP tables were properly cleaned after the reboot. If not, reboot one more time.

### Windows Nodes

To clean up a Windows node, run the script in `c:\etc\rancher`. This script deletes Kubernetes-generated resources and the execution binary. It also drops the firewall rules and network settings:

```
pushd c:\etc\rancher
.\cleanup.ps1
popd
```

After you run this script, the node is reset and can be re-added to a Kubernetes cluster.

### Docker Containers, Images, and Volumes

> Be careful when cleaning up Docker containers. The following command will remove *all* Docker containers, images, and volumes on the node, including non-Rancher related containers:

Reviewer: ```suggestion
:::caution

Be careful when cleaning up Docker containers. The following command will remove *all* Docker containers, images, and volumes on the node, including non-Rancher related containers:

:::
```

<Tabs>
<TabItem value="RKE1">

Before you run the following commands, first remove the node through the Rancher UI.

To remove a node:

1. Click **☰** and select **Cluster Management**.
1. In the table of clusters, click the name of the cluster the node belongs to.

> 1. In the **Machine Pool** tab, click the checkbox next to the node's state.

Reviewer 2: in dashboard, this tab is just `machines`

Reviewer 2: will the UI automatically fix 1. 1. 1. -> 1. 2. 3. ?

Author: @slickwarren I doublechecked, and my instructions lead me to this page, which has the tab labeled "Machine Pools". Are there different places where users can remove nodes? 

![image](https://github.com/rancher/rancher-docs/assets/19174201/b47cc37d-a9cb-45e6-863e-36e5d780931a)

Reviewer 2: are you on 2.7? For custom clusters, the page looks like this for me on 2.7-head for an rke1 cluster
![Screen Shot 2023-06-02 at 11 39 54 AM](https://github.com/rancher/rancher-docs/assets/16691014/cf3b4986-81ca-43c6-9cc6-081d3cc3e032)

Author: Yep, 2.7.3. But I'm looking at the local cluster, not a downstream. Maybe that's it?

Reviewer 2: ah, local cluster is an imported one. afaik, all windows clusters will be custom clusters

Author: With variable tab text, I'm tempted to use "in the first tab" instead of specifying "in the Machine Pools or Machines tab". Do you have any preferences on how to handle this?

Reviewer 2: Honestly, I have no preference. first tab should work too

> 1. In the **Machine Pool** tab, click the checkbox next to the node's state.

Author: ```suggestion
1. In the first tab, click the checkbox next to the node's state.
```
